tests/main/snap-network-errors/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Ensure network errors are handled gracefully
+
+# autopkgtest run only a subset of tests that deals with the integration
+# with the distro
+backends: [-autopkgtest]
+
+# no iptables on core18 yet
+systems: [-ubuntu-core-18-*]
+
+restore: |
+    echo "Remove iptables rules"
+    iptables -D OUTPUT -p udp --dport 53 -j REJECT --reject-with icmp-port-unreachable || true
```

Comment:
I would feel safer if the restore logic loaded a dump of iptables that was made in prepare.

Reply:
Myabe something like:
```
prepare: |
    iptables-save > rules

restore: |
    iptables-restore < rules
```